python/samples/kernel-syntax-examples/chat_gpt_api_function_calling.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+from core_skills_function_call.math_skill import MathSkill
```

Comment:
I didn't want to make changes to the built-in core skills quite yet, but you are right that can be done, I'll have a look. Will also require some work in the setup to disable function calling even though some skills have it (which is needed anyway, since not all ai services will have that ability (both from model and api perspective)